### PR TITLE
feat(v3.6.1) New demo app with SDK v3.6.1

### PR DESCRIPTION
**Pull request by theokir**
- New demo app implementation
- New UI
- Video Ad Support
- Added Links to android-unity-demo app repo and mopub-adapter repo.

### CODE REVIEW

**Comment by emiliosnic** [comment]
Can we remove the extraneous avatars from `app/src/main/assets/feed_avatar_*.png` ?

File: app/src/main/java/com/avocarrot/avocarrotdemoapp/fragments/GameExampleFragment.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view= inflater.inflate(R.layout.fragment_game_example, container, false);
+        ButterKnife.bind(this, view);
+
+       // game_notification_board = (RelativeLayout) view.findViewById(R.id.game_notification_board);
```
Comment (tsiougkosn) [inline app/src/main/java/com/avocarrot/avocarrotdemoapp/fragments/GameExampleFragment.java]: we need that comment ?

**Comment by theokir** [comment]
@emiliosnic On it

File: app/src/main/java/com/avocarrot/avocarrotdemoapp/fragments/ListExampleFragment.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    @Bind(R.id.swipeContainer)
+    SwipeRefreshLayout swipeContainer;
+
+
+    //Initialize click listener for RecyclerView
+    private  ListViewHolder.DemoListItemListener demoListItemListener=new ListViewHolder.DemoListItemListener() {
+        @Override
+        public void onDemoListItemClicked(DemoListItem demoListItem) {
+            Toast.makeText(getContext(),"Demo List Item clicked",Toast.LENGTH_SHORT).show();
```
Comment (tsiougkosn) [inline app/src/main/java/com/avocarrot/avocarrotdemoapp/fragments/ListExampleFragment.java]: Can we show the name of the item that clicked? to demostrate that they need to use avocarrotRecyclerView to get that object

**Pull request merged**